.gitignore

```diff
@@ -16,6 +16,7 @@ npm-debug.log
 .history
 wepback-report.html
 **/yarn-error.log
+yarn.lock
```

Comment:
we wanna ignore yarn.lock, in case some falton try to do `yarn` and then commit

Reply:
sorry, faltong

Reply:
seh, that's the reason i'm adding it to the gitignore